wp-content/themes/rivard-report/functions.php

```diff
@@ -178,7 +178,32 @@ function rivard_popmake_js() {
 			null,
 			true
 		);
-		error_log(var_export( 'much wow', true));
 	}
 }
 add_action( 'wp_enqueue_scripts', 'rivard_popmake_js' );
+
+function rr_google_analytics() {
+	if ( ! is_user_logged_in() ) { // don't track logged in users ?>
+		<script>
+			( function ( i, s, o, g, r, a, m ) {i['GoogleAnalyticsObject']=r;i[r]=i[r]|| function() {( i[r].q=i[r].q||[] ).push( arguments )},i[r].l=1*new Date();a=s.createElement( o ), m=s.getElementsByTagName( o )[0];a.async=1;a.src=g;m.parentNode.insertBefore( a, m )} )
+			( window,document,'script','https://www.google-analytics.com/analytics.js','ga' );
```

Comment:
Reviewing this on vagrant, it looks like we're running both Google Analytics and Universal Analytics on the site at once. Is that something we're worried about?

Reply:
We're only worried if there are 2 or more instances of the old Google Analytics tracking code. Multiple Universal codes and one of the previous kind of code is fine.